app/controllers/commodities_controller.rb

```diff
@@ -14,12 +14,14 @@ def show
 
   private
 
+  attr_reader :commodities
```

Comment:
I don't think we have this getter pattern in any other controller. Not sure it adds value and goes against the grain of other controller implementations

We have attr_reader in these classes:

```
app/presenters/measure_presenter.rb|2 col 3| attr_reader :measure
app/presenters/additional_code_search_presenter.rb|2 col 3| attr_reader :search_form, :search_result, :with_errors
app/presenters/footnote_search_presenter.rb|2 col 3| attr_reader :search_form, :search_result, :with_errors
app/presenters/search_references_presenter.rb|2 col 3| attr_reader :search_references
app/presenters/quota_search_presenter.rb|2 col 3| attr_reader :search_form, :search_result, :with_errors
app/presenters/section_presenter.rb|2 col 3| attr_reader :section
app/presenters/change_presenter.rb|2 col 3| attr_reader :change
app/presenters/chemical_search_presenter.rb|2 col 3| attr_reader :search_form, :search_result, :with_errors
app/presenters/search_reference_presenter.rb|2 col 3| attr_reader :search_reference
app/presenters/certificate_search_presenter.rb|2 col 3| attr_reader :search_form, :search_result, :with_errors
app/presenters/chapter_presenter.rb|2 col 3| attr_reader :chapter
app/presenters/declarable_presenter.rb|2 col 3| attr_reader :declarable
app/models/search/reference_match.rb|13 col 11| array_attr_reader :sections, :chapters, :headings, :commodities
app/models/search/base_match.rb|10 col 17| def array_attr_reader(*args)
app/models/search/goods_nomenclature_match.rb|11 col 11| array_attr_reader :sections, :chapters, :headings, :commodities
app/models/legal_act.rb|14 col 3| attr_reader :validity_start_date,
lib/api_entity.rb|21 col 5| attr_reader :attributes
app/models/search.rb|6 col 3| attr_reader   :q        # search text query
app/models/cookies/policy.rb|5 col 5| attr_reader :usage, :remember_settings
```

Reply:
well, you know, it's a good practice to hide instance variables, but I understand there is not a big gain here